public/javascripts/script.js

```diff
@@ -1,4889 +0,0 @@
-const checkedFilters = {
-  easy: false,
-  features: false,
-  customerSupport: false,
-  valueForMoney: false,
-  distribution: false
-};
-
-document.getElementById("featureEasy").addEventListener("click", () => {
-  if (!checkedFilters.easy) {
-    checkedFilters.easy = true;
-  } else {
-    checkedFilters.easy = false;
-  }
-  document.getElementById("featureEasy").classList.toggle("bg-blue-100");
-  document.getElementById("featureEasy").classList.toggle("border-blue-600");
-  document.getElementById("featureEasy").classList.toggle("border-solid");
-  document.getElementById("featureEasy").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureFeature").addEventListener("click", () => {
-  if (!checkedFilters.features) {
-    checkedFilters.features = true;
-  } else {
-    checkedFilters.features = false;
-  }
-  document.getElementById("featureFeature").classList.toggle("bg-blue-100");
-  document.getElementById("featureFeature").classList.toggle("border-blue-600");
-  document.getElementById("featureFeature").classList.toggle("border-solid");
-  document.getElementById("featureFeature").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureSupport").addEventListener("click", () => {
-  if (!checkedFilters.customerSupport) {
-    checkedFilters.customerSupport = true;
-  } else {
-    checkedFilters.customerSupport = false;
-  }
-  document.getElementById("featureSupport").classList.toggle("bg-blue-100");
-  document.getElementById("featureSupport").classList.toggle("border-blue-600");
-  document.getElementById("featureSupport").classList.toggle("border-solid");
-  document.getElementById("featureSupport").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureValue").addEventListener("click", () => {
-  if (!checkedFilters.valueForMoney) {
-    checkedFilters.valueForMoney = true;
-  } else {
-    checkedFilters.valueForMoney = false;
-  }
-  document.getElementById("featureValue").classList.toggle("border-blue-600");
-  document.getElementById("featureValue").classList.toggle("bg-blue-100");
-  document.getElementById("featureValue").classList.toggle("border-solid");
-  document.getElementById("featureValue").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureDistribution").addEventListener("click", () => {
-  if (!checkedFilters.distribution) {
-    checkedFilters.distribution = true;
-  } else {
-    checkedFilters.distribution = false;
-  }
-  document.getElementById("featureDistribution").classList.toggle("bg-blue-100");
-  document.getElementById("featureDistribution").classList.toggle("border-blue-600");
-  document.getElementById("featureDistribution").classList.toggle("border-solid");
-  document.getElementById("featureDistribution").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-const runFilters = (checkedFilters) => {
```

Comment:
I guess you know what I'm going to say here 😄 

Ok so, really difficult to read and figure out what's going on here. It's super important you write your code in a way that's readable.

Line breaks, variable naming, breaking large functions into smaller ones, etc all help in readability